common/traits/wc_lifestyle_traits.txt

```diff
@@ -72,4 +72,32 @@ harvest_warlock = {
 	random = no
 	ruler_designer_cost = 0
 	customizer = no
+}
+brewmaster = {
+	potential = {
+		OR = {
+			culture_group = vrykul_group
+			culture_group = ogre_group
+			culture_group = dwarf_group
+			culture_group = pandaren_group
+			trait = brewmaster
+		}
+	}
+	lifestyle = yes
+
+	opposites =  {
+	}
+
+	diplomacy = 2
+	drunkard_opinion = 25
+	vrykul_group_opinion = 10
+	ogre_group_opinion = 10
+	dwarf_group_opinion = 10
+	pandaren_group_opinion = 10
```

Comment:
Not localized.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103139956-7a729f80-46fb-11eb-928f-a371c4d0e11d.png)

</details>